### PR TITLE
update access for GH OIDC

### PR DESCRIPTION
**Pull request by zaro0508**
we are switching from using a service user in GH action to 
using a OIDC provider thus we need to provide the OIDC
role equivalent access as the service user role.

### CODE REVIEW

File: config/infra-dev/nextflow-aurora-mysql.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ parameters:
   AccountAdminArns:
     - {{stack_group_config.sso_admin_role.arn}}
     - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
+    - !stack_output_external github-oidc-nextflow-infra::ProviderRoleArn
```
Comment (ConsoleCatzirl) [inline config/infra-dev/nextflow-aurora-mysql.yaml]: This is referencing the stack in the strides account, but everything under `infra-dev/` should reference the stack in the dev account: `sagebase-github-oidc-workflows-dev-nextflow-infra::ProviderRoleArn`

File: config/infra-prod/nextflow-aurora-mysql.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@ parameters:
   AccountAdminArns:
     - {{stack_group_config.sso_admin_role.arn}}
     - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
+    - !stack_output_external github-oidc-nextflow-infra::ProviderRoleArn
```
Comment (ConsoleCatzirl) [inline config/infra-prod/nextflow-aurora-mysql.yaml]: This is referencing the stack in the strides account, but everything under `infra-prod/` should reference the stack in the prod account: `sagebase-github-oidc-workflows-prod-nextflow-infra::ProviderRoleArn`

**Review by ConsoleCatzirl — CHANGES_REQUESTED** [review]

**Comment by zaro0508** [comment]
woops, good catch

File: config/infra-dev/nextflow-aurora-mysql.yaml (hunk 1 above)
Comment (zaro0508) [reply config/infra-dev/nextflow-aurora-mysql.yaml]: fixed

File: config/infra-prod/nextflow-aurora-mysql.yaml (hunk 2 above)
Comment (zaro0508) [reply config/infra-prod/nextflow-aurora-mysql.yaml]: fixed

**Review by ConsoleCatzirl — APPROVED** [review]

**Pull request merged**